Common/eTaxonomy.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Cliquely
+{
+	[Flags]
```

Comment:
Why Flags?

Reply:
The eTaxonomy enum was already defined, I just moved it to common.

Reply:
But should it be Flags? you refactored it so why don't refactor this as well?
You may not refactor it I don't really care